soufi/finders/photon.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2021 Cisco Systems, Inc. and its affiliates
+# All rights reserved.
+
+# TODO(nic): this and the CentOS finder share a lot of the same "bones".  It
+#  may be worth refactoring them both to use common functionality.
```

Comment:
I would say it might be better to do this sooner rather than later if you can, or it probably won't get done at all. It seems like there's a common base class that could be done?

Reply:
I figured I'd write it first and see how much overlap there is.  Turns out: not much.

Reply:
Yeah it looked like it was and then it wasn't ... :laughing: